### DISCUSSION TITLE
Is it possible to use TKinterModernThemes as a plugin in pygubu-designer?

### DISCUSSION THREAD

**Discussion by empowersarch** [alejandroautalan/pygubu-designer#272] [Q&A]
new pygubu-designer user

I was looking at https://pypi.org/project/TKinterModernThemes/ and the pygub-designer example https://github.com/alejandroautalan/pygubu-designer/tree/master/examples/plugin_TKinterModernThemes 

The naming of the example as plugin_TKinterModernThemes made me think that I should be able to do 
pip install pygubu-designer with what ever the plugin name is and get widgets added to pygubu-designer. Is this possible?  If so, What is the command to install it? If not, is there a good resource about how to use TKinterModernThemes in pygubu-designer?

**Comment by alejandroautalan** [comment]
Hello @empowersarch, thanks for trying pygubu, and sorry for the late response.

The pygubu part of the plugin is ready but not released yet.

You can try it installing pygubu and designer from github:

    pip install TKinterModernThemes
    pip install git+https://github.com/alejandroautalan/pygubu.git
    pip install git+https://github.com/alejandroautalan/pygubu-designer.git

What is missing is some features in the designer to allow configure resources more easy. I will try to finish it in a few weeks.

If you try it, let me know.

Regards
Alejandro A.

**Reply by empowersarch** [reply]
I had a bunch of issues at first, the thing kept crashing. I did pip list and realised I had a bunch of stuff installed that could be causing conflicts. 
 
I deleted and remade my venv, redid your steps above and it worked! Yay! Thanks! I am going to play around with it and see what I can make.
